Mathlib/MeasureTheory/Integral/Bochner.lean

```diff
@@ -1352,7 +1352,7 @@ theorem integral_mono_of_nonneg {f g : α → ℝ} (hf : 0 ≤ᵐ[μ] f) (hgi :
   · rw [integral_non_aestronglyMeasurable hfm]
     exact integral_nonneg_of_ae (hf.trans h)
 
-theorem integral_mono_measure {f : α → ℝ} {ν} (hle : μ ≤ ν) (hf : 0 ≤ᵐ[ν] f)
+theorem integral_mono_measure {f : α → ℝ} {ν : Measure α} (hle : μ ≤ ν) (hf : 0 ≤ᵐ[ν] f)
```

Comment:
Any idea why `(hf : 0 ≤ᵐ[ν] f)` was erroring with `nonexhaustive match` without that change?

Reply:
Yes, because otherwise you have `v : ?T` and `T : Sort ?u`, and `?u` is a metavariable so does not participate in the match.